1603.py

```diff
@@ -0,0 +1,18 @@
+class ParkingSystem:
+
+    def __init__(self, big: int, medium: int, small: int):
+        self.lots = dict()
```

Comment:
Since the keys are of integer type, this `dict` can be replaced by an array `[0, big, medium, small]`.